rmf_traffic_ros2/CMakeLists.txt

```diff
@@ -70,7 +70,15 @@ target_include_directories(rmf_traffic_ros2
 )
 
 ament_export_targets(rmf_traffic_ros2 HAS_LIBRARY_TARGET)
-ament_export_dependencies(rmf_traffic rmf_traffic_msgs rclcpp)
+ament_export_dependencies(
+  rclcpp
+  rmf_traffic
+  rmf_traffic_msgs
+  rmf_fleet_msgs
+  Eigen3
```

Comment:
I'm pretty sure `Eigen3` should be covered transitively by `rmf_traffic`. Is that not the case? In fact we shouldn't even need the `find_package(Eigen3 REQUIRED)` up above.

[Here's the relevant line in `rmf_traffic`.](https://github.com/open-rmf/rmf_traffic/blob/f9b5836e179f52309e9d27c4335196be133b5e04/rmf_traffic/cmake/rmf_traffic-config.cmake.in#L7)

Reply:
True, it seems `find_package` could be removed (haven't tested it though), but even though, since it's also directly used by this package, shouldn't we keep it in the `CMakeLists.txt` for clarity and/or future refactors of this code?

Reply:
Sure, I don't have any objection to keeping it :+1: